src/Testcontainers/Configurations/Modules/Databases/CosmosDbTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,67 @@
+namespace DotNet.Testcontainers.Configurations
+{
+  using System;
+  using System.Globalization;
+  using System.IO;
+  using DotNet.Testcontainers.Builders;
+  using JetBrains.Annotations;
+
+  [PublicAPI]
+  public sealed class CosmosDbTestcontainerConfiguration : TestcontainerDatabaseConfiguration
+  {
+    public const string DefaultCosmosDbApiImage =
+      "mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator";
```

Comment:
Can we pin the version?

Reply:
I think that only "lastest" exists.

Reply:
https://mcr.microsoft.com/v2/cosmosdb/linux/azure-cosmos-emulator/tags/list

"2.11.11-5e1f77c6",
"2.11.11-787c2a71",
"2.11.12-efd4ee75",
"2.11.13-1ef6349a",
"2.11.13-36cf4abc",
"2.11.13-a2eeffac",
"2.11.13-c5c131e3",
"2.11.13-sas",
"2.14.5.1",
"2.14.5.2",
"2.14.5.internal",
"latest",
"mongodb",
"mongodb-debug"

"lastest" should be the safest choice.

Reply:
Ok, thanks for the info.